front/migrations/db/migration_86.sql

```diff
@@ -1,44 +1,30 @@
--- -- This migration is dependant on a backfill script
--- -- The backfill script is: <insert_script>
--- -- run psql with --set=backfilled=1 argument if you have rune the script.
-
-CREATE OR REPLACE FUNCTION perform_migration(backfilled boolean DEFAULT false)
-RETURNS VARCHAR AS $$
-BEGIN
-    IF NOT backfilled THEN
-        RAISE NOTICE 'The backfill script: <insert_script> is required before applying this migation. If you already did it, run psql with --set=backfilled=1 argument.';
-    END IF;
-
 -- Migration created on Sep 23, 2024
 UPDATE "data_source_views"
-SET "editedAt" = "updatedAt"
-WHERE "editedAt" IS NULL;
+SET
+    "editedAt" = "updatedAt"
+WHERE
+    "editedAt" IS NULL;
 
 UPDATE "data_source_views"
 SET
     "editedByUserId" = "data_sources"."editedByUserId",
     "editedAt" = "data_sources"."editedAt"
-FROM "data_sources"
+FROM
+    "data_sources"
 WHERE
     "data_sources"."id" = "data_source_views"."dataSourceId"
     AND "data_source_views"."editedByUserId" IS NULL;
 
-ALTER TABLE "data_source_views" ALTER COLUMN "editedAt" SET NOT NULL;
-ALTER TABLE "data_source_views" ALTER COLUMN "editedAt" DROP DEFAULT;
 ALTER TABLE "data_source_views"
-ALTER COLUMN "editedByUserId"
+ALTER COLUMN "editedAt"
 SET
-  NOT NULL;
+    NOT NULL;
 
-    RETURN 'success';
-END;
-$$ LANGUAGE plpgsql;
-
-\if :{?backfilled}
-   SELECT perform_migration(:'backfilled'::boolean);
-\else
-    \echo '!! Migration was NOT applied !!'
-    \echo 'The backfill script: <insert_script> is required before applying this migation. If you already did it, run psql with --set=backfilled=1 argument.'
-\endif
+ALTER TABLE "data_source_views"
+ALTER COLUMN "editedAt"
+DROP DEFAULT;
 
-DROP FUNCTION perform_migration(boolean);
```

Comment:
what is this for ?

Reply:
See context here: https://dust4ai.slack.com/archives/C050SM8NSPK/p1727091558869199